forwardproxy.go

```diff
@@ -197,6 +199,24 @@ func serveHiddenPage(w http.ResponseWriter, authErr error) (int, error) {
 	return 0, nil
 }
 
+func (fp *ForwardProxy) shouldServePacFile(r *http.Request) bool {
+	if len(fp.pacFilePath) > 0 && r.URL.Path == fp.pacFilePath {
+		return true
+	}
+	return false
+}
+
+const pacFile = `
+function FindProxyForURL(url, host) {
+	return "HTTPS %s";
```

Comment:
Port is given to forwardProxy as a string, and then used as string.
`%s:%s` is a great idea, though.